console: Add recently added config properties (#933)
redpanda-data/documentation: docs/platform/console/reference/config.mdx

          # and/or value (just one will work too)
          # valueProtoType: fake_model.Order
          # keyProtoType: package.Type

>     # By default, the baseDir/root is used to import proto files from. To use

Reviewer: How about:

importPaths is a list of paths from which to import Proto files into Redpanda Console. Paths are relative to the (base?root?) directory.

>     # By default, the baseDir/root is used to import proto files from. To use

Reviewer: How are we defining the difference between base directory and root directory?

Reviewer: Is this configuration only relevant if you set `git.enabled`?

Author: > Is this configuration only relevant if you set git.enabled?

Yes. It would make sense to add for the file system provider as well, but it doesn't exist today.

> How are we defining the difference between base directory and root directory?

Synonyms in that context

>     # By default, the baseDir/root is used to import proto files from. To use
>     # multiple import paths relative to the baseDir, set this configuration.
>     # The behavior is similar to the `-I` flag of the proto cli.
>     # This flag only affects the git provider.

Reviewer: ```suggestion
    # importPaths is a list of paths from which to import Proto files into Redpanda Console.
    # Paths are relative to the root directory.
    # The `git` configuration must be enabled to use this feature.
```

  # jwtSecret is a secret string that signs and encrypts
  # the JSON Web tokens used by the backend for session management.
  jwtSecret: redacted

>   # Cookies in Console are used to store user's session data. The session data

Reviewer: We can simplify this paragraph. How about:

`Console stores users' session data in cookies with no fixed size. Because some browsers enforce maximum size limits on cookies, you can enable useCookieChunking to split a single big cookie into multiple smaller ones. When useCookieChunking is enabled, cookies are kept below 4KiB, which is a maximum size limit that is set by most browsers.`

>   # Cookies in Console are used to store user's session data. The session data
>   # is relatively large because Console does not store the session data in a
>   # database, but uses cookies instead. In most cases a single cookie that can
>   # store up to 4KiB in most browsers is sufficient. If your access or refresh
>   # token is too large, you may exceed this maximum size which is enforced in
>   # browser. You can enable cookie chunking to split a single big cookie into
>   # multiple smaller cookie chunks whose size is below the commonly enforced
>   # limit of 4KiB.

Reviewer: ```suggestion
# Redpanda Console stores users' session data in cookies with no fixed size.
# Because some browsers enforce maximum size limits on cookies, you can enable useCookieChunking 
# to split a single big cookie into multiple smaller ones.
# When useCookieChunking is enabled, cookies are kept below 4KiB, 
# which is a maximum size limit that is set by most browsers.
```

    # Path to YAML file that contains all role bindings
    roleBindingsFilepath:

# Analytics is a config block that configures the telemetry sender.

> # By default sending anonymized usage statistics is enabled. The data is

Author: All grammar improvements are always fine by me. Do you mind pushing them straight into the PR or submitting them as code suggestions so that it's easier to apply? I have several PRs opened here and the effort to apply the feedback is higher than needed otherwise I believe.

> # Analytics is a config block that configures the telemetry sender.
> # By default sending anonymized usage statistics is enabled. The data is
> # sent to Redpanda's analytics ingestion endpoint. Redpanda uses the data to
> # evaluate feature usage.

Reviewer: ```suggestion
# analytics configures the telemetry service that sends anonymized usage statistics to Redpanda.
# Redpanda uses these statistics to evaluate feature usage.
```
